### DISCUSSION TITLE
dft command names

### DISCUSSION THREAD

**Discussion by ThomasGeroudet** [The-OpenROAD-Project/OpenROAD#5420] [General]
Hello @maliberty, @fgaray, I've just seen dft commands, but not tried it yet.
The name of the commands seemed familiar, and I was wondering if it could be better to rename them to avoid potential legal issues such as this one:
https://news.synopsys.com/2016-12-21-Synopsys-Obtains-Injunction-in-Copyright-Suit-Against-ATopTech
In case of renaming, maybe all dft commands could start with dft_* so that it's easy to find them ?

**Comment by fgaray** [comment]
Yeah, changing the names SGTM to avoid this issue.

Do we know what other tools use to name this commands? (Cadence for example). I tried looking online but could not find Cadence's related docs.

dft_insert, dft_scan_replace, dft_preview SGTM

**Comment by maliberty** [comment]
Thanks, we want to avoid any such conflicts.  I am not aware of the proprietary tool names but am fine with a rename to avoid such.
